client/src/components/History/CurrentHistory/HistoryOperations/SelectionStatus.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <b-button-group size="sm">
+        <b-button v-if="hasSelection" variant="link" @click="resetSelection" data-test-id="clear-btn">
+            <span class="fa fa-fw fa-times" />
```

Comment:
Maybe add a tooltip to make it more obvious, although it's pretty obvious for me :)
```suggestion
            <span class="fa fa-fw fa-times" title="Clear selection" />
```